src/status_im/activity_center/core.cljs

```diff
@@ -137,13 +139,11 @@
   (and (some? cursor)
        (not= cursor start-or-end-cursor)))
 
-(def ^:const status-read 1)
```

Comment:
FYI: Even though the endpoint `wakuext_activityCenterNotificationsBy` supports the `read` filter type, the UI only toggles between `all` or `unread`. Supporting the `read` filter type would incur in unnecessary computational cost to reconcile notifications, so I removed it.